modules/apps/object/object-web/src/main/resources/META-INF/resources/object_folders/view_model_builder.jsp

```diff
@@ -10,6 +10,7 @@
 <%
 String backURL = ParamUtil.getString(request, "backURL", String.valueOf(renderResponse.createRenderURL()));
 ObjectDefinitionsDetailsDisplayContext objectDefinitionsDetailsDisplayContext = (ObjectDefinitionsDetailsDisplayContext)request.getAttribute(ObjectWebKeys.OBJECT_DEFINITIONS_DETAILS_DISPLAY_CONTEXT);
+ObjectDefinitionsRelationshipsDisplayContext objectDefinitionsRelationshipsDisplayContext = (ObjectDefinitionsRelationshipsDisplayContext)request.getAttribute(ObjectWebKeys.OBJECT_DEFINITIONS_RELATIONSHIP_DISPLAY_CONTEXT);
```

Comment:
I did not see this type of pattern before, using two different _display contexts_ at the same _jsp_.
But I believe it will not cause demages.

Reply:
Hey @mateussgn , I found a pattern using two display contexts in [asset-list-web/src/main/resources/META-INF/resources/init.jsp](https://github.com/liferay/liferay-portal/blob/0ab2d7fce2c261044150c108fdab2d3d4bfa4e90/modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/init.jsp#L102-L103).
I understood that he did this to reuse code from these two preexisting display contexts.
Another option would be to create something like ObjectFoldersViewModelBuilderDisplayContext class with these two inside.

Reply:
Great information! Thanks for bring it up!